src/components/application_manager/src/application_impl.cc

```diff
@@ -1185,6 +1185,9 @@ const std::string& ApplicationImpl::cloud_app_certificate() const {
 }
 
 bool ApplicationImpl::is_cloud_app() const {
+#if !defined(CLOUD_APP_WEBSOCKET_TRANSPORT_SUPPORT)
+  return false;
+#endif  // CLOUD_APP_WEBSOCKET_TRANSPORT_SUPPORT
```

Comment:
I feel that `else` macros should be included, otherwise we will probably get "dead code" warnings from Coverity scans

Reply:
@jacobkeeler should we have is_cloud_app app function if CLOUD_APP_WEBSOCKET_TRANSPORT_SUPPORT is turned off? I would say that definition of  this function can wrapped in ifdef

Reply:
I'd be inclined to agree with you, but this would require quite a significant amount of `ifdef`s to be added to the project (somewhere around 20). Plus it seems to me that this would be a major API change, since this function would no longer be available when this feature is disabled. I think that just having this function return `false` when cloud apps aren't enabled is fine.